src/Printer/CheckStyle.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Povils\PHPMND\Printer;
+
+use Povils\PHPMND\FileReportList;
+use Povils\PHPMND\HintList;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CheckStyle
+ *
+ * @package Povils\PHPMND\Printer
+ */
+class CheckStyle implements Printer
+{
+    /** @var string */
+    private $outputPath;
+
+    public function __construct(string $outputPath)
+    {
+        $this->outputPath = $outputPath;
+    }
+
+    public function printData(OutputInterface $output, FileReportList $fileReportList, HintList $hintList): void
+    {
+        $output->writeln('Generate checkstyle report output...');
```

Comment:
Not sure if these should be printed to stderr ?

Reply:
I just did find what you pointed :)
https://github.com/squizlabs/PHP_CodeSniffer/issues/1612